src/Message/MessageParser.ts

```diff
@@ -24,7 +24,9 @@ export default function parseMessage(
 			tags = parseTags(token.substr(1));
 		} else if (token[0] === ':') {
 			if (!prefix && !command) {
-				prefix = parsePrefix(token.substr(1));
+				if (isServer && token.substr(1) !== '') ( {
```

Comment:
- You don't seem to understand the isServer flag. It indicates whether the line came from a server or from a client, i.e. if you are the client, you receive messages with isServer = true and send them with isServer = false, and if you are the server, it's the other way around.
- The opening parenthesis seems to be a syntax error.
- I'd rather check something like `token.length > 1` - it should perform better than having to cut the string just for the check.

Reply:
The idea of the isServer check was that if the message was coming from a client `(isServer === false)`, it doesn't need to be parsed, and could be ignored. Removed.

I'm confused how the opening parenthesis is an error, it's working fine in my environment (typescript / eslint / prettier)

token.length > 1 makes sense. Fixed.